### ISSUE TITLE
Documentation for usage with minimal TypeScript setup

### ISSUE THREAD

**Issue by Shinigami92** [faker-js/faker#464]
### Clear and concise description of the problem

Currently it's not clear for a TypeScript developer that they need to configure their `tsconfig.json` with the `compilerOptions` seen below.
We need to document that.

### Suggested solution

```jsonc
// tsconfig.json
{
  "compilerOptions": {
    "esModuleInterop": true,
    "moduleResolution": "Node"
  }
}
```

**Comment by sprohaszka** [comment]
Hi @Shinigami92,

Just to know how what to write, I've just launched a simple project with: typescript, jest and faker.
I started with project with a basic `tsconfig.json` file created when you run `tsc --init`.

The `esModuleInterop` is already set to `true`, but no `moduleResolution` option is set.
According to [typescript documentation](https://www.typescriptlang.org/tsconfig#moduleResolution) the default value is `Classic`.

I can do the modification on the README if you want, but are you sure about the last option?

**Comment by Shinigami92** [comment]
> I can do the modification on the README if you want, but are you sure about the last option?

No I'm not 100% sure, but think I have tested it and it did not work or at least did not use the esm version
Maybe something is fixed now after #520 :thinking: 
But then it would take the cjs instead of esm, so maybe the `moduleResolution` is only valuable if you want to use esm, _and I think you want_

**Comment by sprohaszka** [comment]
Ok :)
I pushed a PR. Let me know if it was what you had in mind.
